src/button-dropdown/category-elements/styles.scss

```diff
@@ -59,20 +59,6 @@
         box-shadow: inset 0 0 0 awsui.$border-control-focus-ring-shadow-spread awsui.$color-border-item-focused;
       }
     }
-
-    &.variant-navigation {
```

Comment:
So this one is supposed to be added when expandable-header is set. However, we didn't support expandable menus in top navigation up until now and the styles were causing incorrect menu item display when highlighted.